### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by exihuatl** [devongovett/atom-jade#32]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (C:\Users\Renia\AppData\Local\atom\app-0.189.0\resources\app\src\package.js:565:9)
<unknown> (C:\Users\Renia\AppData\Local\atom\app-0.189.0\resources\app\src\package.js:187:43)
```

**Comment by thefivetoes** [comment]
This is a duplicate of: https://github.com/devongovett/atom-jade/issues/30

**Comment by LinusU** [comment]
Fixed in 0.4.0
